interfaces/udev/udev.go

```diff
@@ -45,7 +72,7 @@ func (b *Backend) reloadRules(subsystemTriggers []string) error {
 	// By default, trigger for all events except the input subsystem since
 	// it can cause noticeable blocked input on, for example, classic
 	// desktop.
-	output, err = exec.Command("udevadm", "trigger", "--subsystem-nomatch=input").CombinedOutput()
+	output, err = exec.Command("udevadm", maybeTriggerQuiet("--subsystem-nomatch=input")...).CombinedOutput()
```

Comment:
this change is wrong, we need to ignore the error from running this, `--quiet` doesn't change the exit code behavior at all AFAICT, this is evidenced by the fact that lxd spread tests are still failing on this PR

Reply:
I guess if we wanted to, we could make the exit code ignoring behavior conditional on only 249 and newer systemd versions, but I'm not sure what the point of that is, do we have situations where udevadm actually fails and we want to fail something inside snapd because of that?